test/e2e/framework/framework.go

```diff
@@ -105,26 +110,76 @@ func newFramework(kubeconfig string, qps float32, burst int) (*Framework, error)
 	}, nil
 }
 
+const (
+	kubeconfigFlag      = "kubeconfig"
+	gsimageFlag         = "gameserver-image"
+	pullSecretFlag      = "pullsecret"
+	stressTestLevelFlag = "stress"
+	perfOutputDirFlag   = "perf-output"
+	versionFlag         = "version"
+	featureGatesFlag    = "feature-gates"
```

Comment:
Curious - why are we re-declaring `featureGatesFlag`? We could use the const value from utils/runtime no?

Reply:
Yes, we can use that variable. Made it exportable.